biglog/segment.go

```diff
@@ -164,6 +173,9 @@ func loadSegment(indexPath string) (seg *segment, err error) {
 	seg.setNextOffsets()
 	seg.setCreatedTS()
 
+	// TODO: when this is called via createSegment the index is
+	// created by createSegIndex which doesn't write this timestamp?!
```

Comment:
ok that's good to know anyway and clears a couple of questions I had. Still there is either a surprising behaviour or a bug here. Please have a look at the new [`TestCreateSegment_WithoutCreatedTS` test][1] to see what I mean.

[1]: https://github.com/fgrosse/netlog/commit/b8f68df846d8e0e7d4db9a47e33297ba122d3d48